rfc/0016-specification-repository.md

```diff
@@ -0,0 +1,54 @@
+# Summary
+
+This RFC amends the RFC process with the creation of a specifications repository, which holds the body of some policy defined through RFCs here.
+
+This RFC does not seek to change the way policy and documentation is reviewed and accepted, only how it is amended and referenced.
+
+# Motivation
+
+Currently, policy on team structure, voting, and other critical aspects of governance are defined only through RFCs.
+This is an issue when a major part of that RFC needs to be redefined--the entire RFC template is excluded for these changes! This is fine for clarifications, but in large-scale changes that require lots of discussion, the current process works against the point of defining these documents through RFCs in the first place.
+
+
+# Explanation
+
+The `quiltmc/specifications` repository created by this RFC can hold three major types of specifications:
+
+1. File specifications
+2. Governance policy
+3. Technical definitions (see: ASMR, QSL)
+   For simplicity, each of these kinds of documents will be called specifications within this RFC.
+   Each specification would be its own file.
+   Creation or major revisions (to be determined by a case-by-case basis) of a specification requires an RFC.
+   Clarifications and other minor revisions can be made by making a PR directly to the specification repository.
+
+Each specification only contains the actionable parts of the RFCs involved (usually summary and explaination). Motivation, Rationale, Unresolved Questions, etc. are left only on the RFC.
```

Comment:
This whole section reads... oddly. Documents? Sections? Is point three meant to be that huge paragraph? You define "specifications" within point three, but use the terminology before that in the section.

Reply:
Point three is not meant to be that long, fixing that.